### PR TITLE
update rpm page

### PR DESCRIPTION
**Pull request by Hwurzburg**
Co-authored-by: Matt Kear <matthewnkear@gmail.com>

I am splitting your PR into a merge-able one (4.1 things) and will create an upcoming features PR for the commit on the ESC PRM Type ....I will put you as co-author on both...and close your existing one...
added a bit to the base one also...

### CODE REVIEW

File: common/source/docs/common-rpm.rst [hunk 1]
```diff
@@ -66,38 +68,37 @@ and is recommended.
...
 be used for GPIO.  On non-Pixhawk boards the PWM count will include all PWM outputs.  On Pixhawk boards this parameter only affects AUX pins.  Write the parameter 
 and reboot the autopilot.
 
-Now the autopilot needs to be told which pin to find the RPM signal is on.  To do this you will need to find the pin number in the hwdef.dat file for you 
-board.  These can be found `here <https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_HAL_ChibiOS/hwdef>`__.  Within the file for your board you will 
-need to find where the GPIO(*) definition is.  The below example is an excerpt from the hwdef.dat file for the cube: ::
+Now the RPM library must be enabled. In the following sections, we will use the second instance of RPM sensor for parameter examples.
 
-    PD13 TIM4_CH2 TIM4 PWM(5) GPIO(54)
+Set the parameter :ref:`RPM2_TYPE<RPM2_TYPE>` to 1 for a GPIO pin based sensor.  Write the parameter to ArduPilot then refresh/fetch the
```
Comment (Gone4Dirt) [inline common/source/docs/common-rpm.rst:89]: Can the example set RPM2_TYPE to 2 please.  We have left type = 1 active so as not to break any existing setups but the param meta-data does now say "1:Not Used" so we should maintain the theme in the wiki.

File: common/source/docs/common-rpm.rst [hunk 2]
```diff
@@ -66,38 +68,37 @@ and is recommended.
...
 **Parameter Setup**
 
 First the board needs to be configured to allow PWM pins to be set for GPIO.  This is done using the parameter ``BRD_PWM_COUNT`` .  Reduce the PWM count to free up a pin to 
 be used for GPIO.  On non-Pixhawk boards the PWM count will include all PWM outputs.  On Pixhawk boards this parameter only affects AUX pins.  Write the parameter 
 and reboot the autopilot.
 
-Now the autopilot needs to be told which pin to find the RPM signal is on.  To do this you will need to find the pin number in the hwdef.dat file for you 
-board.  These can be found `here <https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_HAL_ChibiOS/hwdef>`__.  Within the file for your board you will 
-need to find where the GPIO(*) definition is.  The below example is an excerpt from the hwdef.dat file for the cube: ::
+Now the RPM library must be enabled. In the following sections, we will use the second instance of RPM sensor for parameter examples.
```
Comment (Gone4Dirt) [inline common/source/docs/common-rpm.rst:87]: Not an issue, but just wondering why we are doing an example on the 2nd instance and not the first?

File: common/source/docs/common-rpm.rst [hunk 3]
```diff
@@ -114,16 +115,21 @@ Electrical commutation RPM sensors can be added retrospectively using something
...
-Again, the setup of optical sensors is much the same as the hall effect sensor.  The only difference is the scaling value.  :ref:`RPM1_SCALING<RPM1_SCALING>` should be set to be the reciprocal 
-of the number of reflective obstacles that pass the sensor in a single rotation. 
+Again, the setup of optical sensors is much the same as the hall effect sensor.  The only difference is the scaling value.  :ref:`RPM2_SCALING<RPM2_SCALING>` should be set to be the reciprocal 
+of the number of reflective obstacles that pass the sensor in a single rotation.
+
+
+Harmonic Notch Center Frequency
+===============================
 
+If ``RPMx_TYPE`` = 4, then the center frequency of the harmonic notch, however it is obtained (See :ref:`common-imu-notch-filtering`), is reported.
```
Comment (Gone4Dirt) [inline common/source/docs/common-rpm.rst:135]: Maybe remove the sub-clause and move it to the end of the sentence?  Took me a couple of goes reading it to understand what was being said. e.g:
```suggestion
If ``RPMx_TYPE`` = 4, then the center frequency of the harmonic notch is reported.  See :ref:`common-imu-notch-filtering` for how this value is obtained.
```

**Review by Gone4Dirt — CHANGES_REQUESTED** [review]
Thanks for doing this @Hwurzburg.

File: common/source/docs/common-rpm.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-rpm.rst:89]: done

File: common/source/docs/common-rpm.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-rpm.rst:87]: just to highlite that there are more than one RPM instance available

**Pull request merged**